RecoTracker/MeasurementDet/BuildFile.xml

```diff
@@ -7,7 +7,6 @@
 <use name="DataFormats/SiPixelCluster"/>
 <use name="FWCore/Framework"/>
 <use name="FWCore/ParameterSet"/>
-<use name="Geometry/TrackerGeometryBuilder"/>
```

Comment:
used in RecoTracker/MeasurementDet/plugins, but not present in the BuildFile there already.